src/chapters/angular-wk1/templates.rst

```diff
@@ -0,0 +1,176 @@
+Templates
+==========
+Take a look at the homepage for `LaunchCode <https://www.launchcode.org/>`__.
+The content includes text, images, a navigation bar, buttons, embedded videos,
+static and scrolling sponsor logos, and links. All of this content is carefully
+arranged.
+
+Your Own Website
+-----------------
+
+Imagine building your own website with a smaller set of features. You could
+include favorite movies or sports teams, cities where you have lived, schools
+attended, hobbies, etc. Using your :ref:`HTML <html-index>` and
+:ref:`CSS <css-index>` knowledge, you could construct an appealing layout using
+carefully selected and arranged tags (``<div>, <button>, <img>``, etc.).
+
+Now imagine you have to change one or more items on the page. Maybe you move
+and your school or team loyalty shifts. Updating the city, school logo, etc.
+means adjusting those items in the HTML, and if that information appears in
+other areas of your website, then you need to update that code as well. Also,
+you need to consider any formatting changes that result from adding or removing
+content.
+
+Whew! Updating your website rapidly gets tedious, especially if it contains
+lots of information or consists of multiple pages. If only there was a way to
+automatically update the content and layout for your site...
+
+Templates are Frameworks
+-------------------------
+
+.. index:: ! template
+
+A **template** provides the general structure for a web page. It identifies
+where different elements get placed on the page, but it does NOT fill them with
+content. Think of a template like a skeleton, a blueprint, or an outline for
```

Comment:
I personally don't like using multiple analogies (skeleton, blueprint, or an outline). I personally would say pick one, and go with that. But that's just my opinion if you feel strongly about using all three feel free to leave them in.

Reply:
I kept my favorite.